ci-operator/step-registry/telcov10n/functional/cnf-ran/cnf-tests-sno-worker/telcov10n-functional-cnf-ran-cnf-tests-sno-worker-commands.sh

```
@@ -0,0 +1,193 @@
+    node-role.kubernetes.io/workercnf="
+
+cd /eco-ci-cd
+
+echo "Run ansible playbook to generate CNF test scripts"
+ansible-playbook ./playbooks/cnf/deploy-run-cnf-tests-script.yaml \
+  -i ./inventories/cnf/switch-config.yaml \
+  --extra-vars "kubeconfig=${SPOKE_KUBECONFIG} features=${CNF_TESTS_FEATURES} cnf_test_dir=${CNF_TEST_DIR}" \
+  -vv
+
+echo "Fix ginkgo version alignment and run CNF tests via SSH"
+ssh ${SSH_OPTS_KEEPALIVE} "${BASTION_USER}@${BASTION_IP}" -i "${PROJECT_DIR}/temp_ssh_key" bash -s <<'REMOTE_SCRIPT'
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)